app/src/main/java/com/example/drawingactivity/CurrentColorView.kt

```diff
@@ -17,7 +17,7 @@ import android.view.View
  * circlePrimaryColor is how the color of the circle is adjusted
```

Comment:
Remember to include your ticket number in your PR name and your commit name

Reply:
Will remember to do this going forward